Lib/asyncio/sslproto.py

```diff
@@ -381,8 +381,13 @@ def write(self, data):
         if not isinstance(data, (bytes, bytearray, memoryview)):
             raise TypeError(f"data: expecting a bytes-like instance, "
                             f"got {type(data).__name__}")
+        if self._ssl_protocol._sslpipe is None:
+            # The SSL pipe has been destroyed. Writing should fail.
+            # See bpo-25292.
+            raise BrokenPipeError()
```

Comment:
I think an error message describing the error might be helpful to the user. Maybe something like
`BrokenPipeError("SSL pipe is broken")`   ?

Reply:
Thats good. I added your suggested error message